demos/apps/bookinfo/cleanup.sh

```diff
@@ -25,4 +25,5 @@ istioctl delete route-rule ratings-test-delay
 #istioctl delete mixer-rule ratings-ratelimit
 
 kubectl delete -f $SCRIPTDIR/bookinfo-istio.yaml
-kubectl delete -f $SCRIPTDIR/../../istio
+kubectl delete -f $SCRIPTDIR/../../ingress-controller.yaml
```

Comment:
If we use a non-default namespace, we can simply delete the whole namespace.
Alternatively, we can use commands like:
`kubectl delete deployments --all`
`kubectl delete services --all`
`kubectl delete configmap mixer`
`kubectl delete ingress --all`